lib/animas/animasSimulator.js

```diff
@@ -85,9 +90,53 @@ exports.make = function(config){
     events.push(e);
   }
 
+  function checkIfBasalSuspendedByAlarm(alarm,basal) {
```

Comment:
Since this function mutates the basal, can we call it something else? `checkIfBasalSuspendedByAlarm` made me think it was going to return `true` or `false`. Maybe `markIfBasalSuspendedByAlarm`?